internal/global/trace.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"sync/atomic"
 
+	"go.opentelemetry.io/auto/sdk"
```

Comment:
importing an "SDK" seems wrong from our API (I haven't looked at the package).  Is the naming just misleading?  Or do we need to split it into api vs sdk?

Reply:
Maybe the naming is just misleading. It is an implementation of the OTel API, similar to what exists in this package, so it is called `sdk`.

Reply:
There is no OTel API in that package as it does not provide its own API. Similar to this global package.

Reply:
Got it.  At least it doesn't bring in any additional dependencies.  So the only potential concern would be binary size, and this SDK doesn't look that large.

Reply:
Correct :+1: . It is designed to not have any more dependencies than what the `otel` package already contains.

Reply:
I agree the name is misleading. How about importing this as `auto`?

```suggestion
	auto "go.opentelemetry.io/auto/sdk"
```

Reply:
I don't want to give the impression we are using `go.opentelemetry.io/auto` here.

I think given this is just a name it is not significant to change this import name.